.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,58 @@
+// Dev container notes for developers:
+//
+// You do not need to use a dev container with matter.js.  Native development should work fine.  Further, due to
+// networking idiosyncrasies, devices running in a dev container may not be discoverable outside the container on macs
+// and windows.
+//
+// For the same reasons as above, the dev container is useful for local testing.  Running matter.js in the container
+// allows it to connect to other components running in docker.
+//
+// Using Docker desktop you must set default containers to be dual IPv4/IPv6.  On macs the options "Use kernel
+// networking for UDP" and "Enable host networking" must be enabled.
+
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+	"name": "matter.js",
+	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+	"image": "mcr.microsoft.com/devcontainers/typescript-node:1-22-bookworm",
+
+	"postCreateCommand": "npm install",
```

Comment:
Is npm ci better to ensure same deps?

Reply:
good point, really I should never use npm install, just ci and relock